rmf_traffic_ros2/src/rmf_traffic_ros2/schedule/MirrorManager.cpp

```diff
@@ -70,13 +75,17 @@ class MirrorManager::Implementation
 
   rclcpp::Node& node;
   rmf_traffic::schedule::Query query;
+  uint64_t query_id = 0;
+  bool require_query_validation = false;
+  uint64_t expected_node_edition = 0;
```

Comment:
I initially used `version` but I felt very uncomfortable with the clash with other uses of the word. I also feel that it does not reflect what it actually is: it's not a different version of the schedule node.

Having thought more on it, would `generation` work better?

Reply:
I see where you're coming from with `generation`, but I'm concerned it's too overloaded of a word. I think in the context of programming it's more often used to refer to the "act of generating".

I don't have a super strong preference here because I think there are fair arguments to be made for any of the choices, but I do think we should be deliberate about what we choose, because the wording we go with right now could influence formal design docs and specifications in the future. What if we do a quick internal poll/discussion to see if there's any obvious preference when we take more people's views into account?

Reply:
A quick internal poll/discussion sounds like a good idea to me.

Reply:
The poll was inconclusive, so I will change it to `version` to be consistent.

Reply:
Done in e4b5879.